tmt/steps/provision/local.py

```diff
@@ -133,6 +133,43 @@ def pull(
             extend_options: Optional[list[str]] = None) -> None:
         """ Nothing to be done to pull workdir """
 
+    def acquire_log(self, log_name: str) -> Optional[str]:
+        """fetch and return content of a requested log"""
+        if log_name == 'dmesg':
+            return self.execute(Command('dmesg')).stdout
+        return None
+
+    def store_log(
+            self,
+            log_name: Optional[str],
+            log_path: Optional[Path],
+            log_content: str) -> None:
+        """save log content and return the path"""
+        if log_path and log_name:
+            # if log_path contains log file name
+            if log_path.name == log_name:
```

Comment:
`if log_path.is_dir` would be better - what if the caller gives us a filename that does not match the log name? It's not up to us and here to dispute that.

Reply:
updated^^